perlvars.h

```diff
@@ -265,7 +265,8 @@ PERLVARA(G, hash_seed_w, PERL_HASH_SEED_WORDS, PVT__PERL_HASH_WORD_TYPE) /* perl
 PERLVARA(G, hash_state_w, PERL_HASH_STATE_WORDS, PVT__PERL_HASH_WORD_TYPE) /* perl.c and hv.h */
 #endif
 #if defined(PERL_USE_SINGLE_CHAR_HASH_CACHE)
-PERLVARA(G, hash_chars, (1+256) * sizeof(U32), unsigned char) /* perl.c and hv.h */
+#define PERL_SINGLE_CHAR_HASH_CACHE_ELEMS ((1+256) * sizeof(U32))
+PERLVARA(G, hash_chars, PERL_SINGLE_CHAR_HASH_CACHE_ELEMS, unsigned char) /* perl.c and hv.h */
 #endif
```

Comment:
Similar to the above question ... Is `PERL_SINGLE_CHAR_HASH_CACHE_ELEMS` going to be used anyplace other than in the line immediately following its define? If not, what is the rationale for this change?

Reply:
I thought I did explain in the commit message: it makes it easier to parse the file. The secondary answer is that it is always better to hide a complex constant expression behind a named identifier, it adds comprehensibility and it allows the complex expression to be reused. 

Indeed, I could easily reuse this define in the hash code, I am fairly sure I have duplicated this logic twice. 

BTW, check the blame log for added this line of code in the first place. Arguably I should have used a define in the first place! ;-)

Reply:
@jkeenan ping.

Reply:
As above.